File: MySolutionExplorer/Templates/example 0000. Template [cs]/Task_example0000.cs
using System;
using System.IO;
using System.Collections.Generic;

public class Task
{
	public static void Main()
	{
		#region Settings

		StreamReader In;
		StreamWriter Out;
		try
		{
			System.Threading.Thread.CurrentThread.CurrentCulture =
				System.Globalization.CultureInfo.InvariantCulture;
		}
		finally
		{
#if file
			Directory.SetCurrentDirectory(new DirectoryInfo(Directory.GetCurrentDirectory()).Parent.FullName);
			In = new StreamReader("input.txt");
			Out = new StreamWriter("output.txt");
#else
			In = new StreamReader(Console.OpenStandardInput());
			Out = new StreamWriter(Console.OpenStandardOutput());
#endif
		}

		#endregion

		var s = In.ReadLine().Split();
		var a = int.Parse(s[0]);
		var b = int.Parse(s[1]);
		Out.WriteLine(a + b);

		Out.Close();
	}
}

File: MySolutionExplorer/Templates/example 0000. Template [java] - копия/src/Task_example0000.java
import java.io.*;
import java.util.*;

class Task {
    private static Scanner In;
    private static PrintWriter Out;

    public static void main(String[] args) throws Exception {
        if (args.length > 0) {
            In = new Scanner(new File("input.txt"));
            Out = new PrintWriter(new File("output.txt"));
        }
        else {
            In = new Scanner(System.in);
            Out = new PrintWriter(System.out);
        }

        String s[] = In.nextLine().split(" ");
        int a = Integer.parseInt(s[0]);
        int b = Integer.parseInt(s[1]);
        Out.println(a + b);

        Out.close();
    }
}
